client/components/domains/domain-product-price/index.jsx

```diff
@@ -93,8 +102,8 @@ const DomainProductPrice = React.createClass( {
 } );
 
 export default connect(
-	state => ( { domainsWithPlansOnly: getCurrentUser( state ) ?
-		currentUserHasFlag( state, DOMAINS_WITH_PLANS_ONLY ) :
-		true
+	state => ( { domainsWithPlansOnly: getCurrentUser( state )
+		? currentUserHasFlag( state, DOMAINS_WITH_PLANS_ONLY )
+		: true
```

Comment:
While I like reduction to avoid ternaries, I think readability trumps terseness. IMO keep the ternary for clarity.